### ISSUE TITLE
Mashups info window cropped when marker is too close to top of frame

### ISSUE THREAD

**Issue by kevinwhoffman** [WordImpress/google-maps-builder#108]
When clicking a marker near the top of the frame, the map repositions itself but not far enough to contain the entirety of the info window. I think what is happening is the reposition occurs before all of the content is loaded, so it doesn't know how far north it needs to move in order to reveal the entire info window. See below:

![info-window-cropped-1](https://cloud.githubusercontent.com/assets/1938671/9533929/2f4155be-4ce3-11e5-9801-b9849618fb22.jpg)

At this point the reposition has already occurred, so when the content loads in a second later, it becomes cropped.

![info-window-cropped-2](https://cloud.githubusercontent.com/assets/1938671/9533931/34167e48-4ce3-11e5-8bbb-801ce60154df.jpg)

Can you wait to fire the reposition until after the content has loaded?

**Comment by DevinWalker** [comment]
Yeah I noticed that too. Should it center on the map or just scroll into view?

**Comment by kevinwhoffman** [comment]
I prefer centering it. With that said, I don't think centering is as important at far out zoom levels, but at closer zoom levels, centering the marker allows you to see more of the surrounding context. You can see that behavior here: http://celebrateditaly.com/map/

**Comment by kevinwhoffman** [comment]
Centering also better ensures that the info window doesn't overlap controls when clicking a marker along the edge of the map.
